solvebio/cli/main.py

```diff
@@ -93,8 +104,7 @@ class SolveArgumentParser(argparse.ArgumentParser):
                 },
                 {
                     'name': 'dataset',
-                    'help': 'The full name of the dataset '
-                            '(<depository>/<version>/<dataset>)'
+                    'help': 'The name of the dataset'
```

Comment:
When you get the object back, the path will include the filename, but I was thinking that for the creation, the `--path` value that the user provides would be the parent directory that the dataset will be placed into.  Does this feel awkward, or OK?  Some examples:

```
# Create a dataset at root
solvebio create-dataset --path=/ --name=new-dataset --vault=my-vault

# Create dataset at /ngs-output
solvebio create-dataset --path=/ngs-output --name=new-dataset --vault=my-vault
```